.github/workflows/ci-testing.yml

```diff
@@ -93,7 +93,7 @@ jobs:
               --ignore=thunder/tests/distributed \
               -v --datefmt="%Y%m%d-%H:%M:%S.%f" \
               --random-order-seed=$GITHUB_RUN_ID \
-              -n 2 --durations=250
+              -n 4 --durations=250
```

Comment:
so the runners for linux and win have both 4 CPUs, increasing this to 4 make Windows go from ~28 to ~22 and Linux is now slowest at ~23:30 (before the trick for jvp).